helm/app/values.yaml

```diff
@@ -1,3 +1,21 @@
+namespace: go-api-boilerplate
+
+rbac:
+  # Specifies whether rbac resources must be created.
+  create: true
+  # The API version to use while creating the rbac resources. Use `kubectl api-versions | grep rbac` to find which abi versions are supported for your cluster.
+  apiVersion: v1
+  # Create a cluster role binding with the existing role, do not create a new one. If left blank, a new cluster role is created.
+  roleRef: admin
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # The name of the ServiceAccount to use.
+  # If not set and create is true, a name is generated using the fullname template
+  # If create is false and a name is not specified, the default service account is used for the cluster role binding.
+  name: goapiboilerplate
```

Comment:
should be probably something like this or similar

```
name: {{ include "app.fullname" $ }}
```

Reply:
Yes, since there is a default value we can leave it blank and easily adjustable globally if needed: `{{ .Values.serviceAccount.name | default (printf "%s" (include "app.fullname" $))}}`